src/components/PathwayCourse/index.js

```diff
@@ -88,35 +144,115 @@ function PathwayCourse() {
   const pathwayCourseData = pathways.find((item) => {
     return item.id == pathwayId;
   });
-
+  useEffect(() => {
+    console.log(
+      upcomingBatchesData,
+      userEnrolledClasses,
+      pathwayCourse?.data?.courses,
+      "Here"
+    );
+  }, [upcomingBatchesData, userEnrolledClasses]);
   return (
     <>
+      {enrolledBatches ? (
+        <>
+          <Typography
+            sx={{
+              fontWeight: "bold",
+              justifyContent: "center",
+              height: "40px",
+              display: "flex",
+              alignItems: "center",
+            }}
+            bgcolor="#E9F5E9"
+          >
+            {enrolledBatches[0]?.title}
+          </Typography>
+        </>
+      ) : (
+        ""
+      )}
+
       <Container className={classes.pathwayContainer} maxWidth="lg">
-        {pathwayId && pathwayCourseData && (
+        {enrolledBatches ? (
           <>
-            <Grid container spacing={2} align="center" className={classes.box}>
-              <Grid xs={12} md={6}>
-                <Card align="left" elevation={0} className={classes.titleCard}>
-                  <Typography
-                    variant="body2"
-                    className={classes.cardSubtitle}
-                    sx={{ textAlign: isActive && "center", pb: "8px" }}
-                  >
-                    Learning Track
-                  </Typography>
-                  <Typography
-                    variant="h4"
-                    className={classes.heading}
-                    sx={{ textAlign: isActive && "center", pb: "16px" }}
+            <PathwayCards userEnrolledClasses={userEnrolledClasses} />
+          </>
+        ) : (
+          pathwayId &&
+          pathwayCourseData && (
+            <>
+              <Grid
+                container
+                spacing={2}
+                align="center"
+                className={classes.box}
+              >
+                <Grid item xs={12} md={6} sx={{ pl: 2 }}>
+                  <Card
+                    align="left"
+                    elevation={0}
+                    className={classes.titleCard}
                   >
-                    {pathwayCourseData.pathway}
-                  </Typography>
-                  <Typography variant="body1">
-                    {pathwayCourseData.description}
-                  </Typography>
-                </Card>
-              </Grid>
-              {/* <Grid xs={12} md={6} sx={{ pl: 2 }}>
+                    <Typography
+                      variant="body2"
+                      className={classes.cardSubtitle}
+                      sx={{ textAlign: isActive && "center", pb: "8px" }}
+                    >
+                      Learning Track
+                    </Typography>
```

Comment:
Use `ResponsiveTypography` component here, and then you can remove `textAlign: isActive && "center"`